keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -349,19 +353,6 @@ Contains a unique identifier for the containing cluster.
 
 **Reusing cluster names**: Since an `id.k8s.io ClusterClaim` has no restrictions on whether or not a ClusterClaim can be repeatable, if a cluster unregisters from a ClusterSet it is permitted under this standard to rejoin later with the same `id.k8s.io ClusterClaim` it had before. Similarly, a *different* cluster could join a ClusterSet with the same `id.k8s.io ClusterClaim` that had been used by another cluster previously, as long as both do not have membership in the same ClusterSet at the same time. Finally, two or more clusters may have the same `id.k8s.io ClusterClaim` concurrently (though they **should** not; see "Uniqueness" above) *as long as* they both do not have membership in the same ClusterSet.
 
-```
-<<[UNRESOLVED]>>
-We could probably use some example scenarios describing how some dependent tool should handle IDs changing or disappearing during various stages of the cluster/membership lifecycle - @jeremyot
-<<[/UNRESOLVED]>>
-```
-
-```
-<<[UNRESOLVED]>>
```

Comment:
This is addressed below at https://github.com/kubernetes/enhancements/pull/2408/files#diff-8bc62b436dee634db23bfcfd2388e6a38ac9edff8c481e6347f873765a7bae7fR455 and https://github.com/kubernetes/enhancements/pull/2408/files#diff-8bc62b436dee634db23bfcfd2388e6a38ac9edff8c481e6347f873765a7bae7fR482